### PR TITLE
Accessories Page

### PR DESCRIPTION
**Pull request by SaiLaxmiRaj**
Please,Review my code.
I add one test case under Apparel page.

### CODE REVIEW

File: src/main/java/com/nopcommerce/demo/pages/ApparelPage.java [hunk 1]
```diff
@@ -33,6 +33,14 @@ public ApparelPage() {
     @FindBy(tagName = "h1")
     WebElement verifyTextClothing;
 
+    @CacheLookup
+    @FindBy(xpath = "//a[@title='Show products in category Accessories'][normalize-space()='Accessories']")
+    WebElement accessoriesLink;
+
+    @CacheLookup
+    @FindBy(tagName = "h1")
```
Comment (HiteshVagQa) [inline src/main/java/com/nopcommerce/demo/pages/ApparelPage.java]: Can you use other strong locator technique

File: src/main/java/com/nopcommerce/demo/pages/ApparelPage.java [hunk 2]
```diff
@@ -61,4 +69,19 @@ public void setVerifyTextClothingText()
         Assert.assertEquals(actualMessage,expectedMessage);
```
Comment (HiteshVagQa) [inline src/main/java/com/nopcommerce/demo/pages/ApparelPage.java:82]: Please add message to show if assert fail
i.e. should link not found

**Review by HiteshVagQa — CHANGES_REQUESTED** [review]

File: src/main/java/com/nopcommerce/demo/pages/ApparelPage.java [hunk 3]
```diff
@@ -61,4 +69,19 @@ public void setVerifyTextClothingText()
...
+        log.info("MouseHover On Accessories " + accessoriesLink.toString());
+        mouseHoverToElementAndClick(accessoriesLink);
+    }
+
+    public void setVerifyTextAccessories()
+    {
+        String expectedMessage = "Accessories";
+        String actualMessage = getTextFromElement(verifyTextAccessories);
+        Assert.assertEquals(actualMessage,expectedMessage);
+        log.info("Verify text Accessories " + verifyTextAccessories.toString());
```
Comment (ghost) [inline src/main/java/com/nopcommerce/demo/pages/ApparelPage.java]: Can you change text to Text

Comment (SaiLaxmiRaj) [reply src/main/java/com/nopcommerce/demo/pages/ApparelPage.java]: which "text"  need to  change it i didn't get it.

File: src/test/java/resources/featurefile/apparelpage.feature [hunk 4]
```diff
@@ -15,3 +15,8 @@ Feature: ApparelMenu Test
     Scenario: User should navigate to clothing page successfully
```
Comment (ghost) [inline src/test/java/resources/featurefile/apparelpage.feature:15]: change clothing page starting with capital latter

**Review by HiteshVagQa — APPROVED** [review]

File: src/main/java/com/nopcommerce/demo/pages/ApparelPage.java (hunk 3 above)
Comment (SaiLaxmiRaj) [reply src/main/java/com/nopcommerce/demo/pages/ApparelPage.java]: I resolved it. Just check for me
Thank you

**Pull request merged**